decidim-templates/app/views/decidim/templates/admin/proposal_answer_templates/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<% add_decidim_page_title(t(".title")) %>
+
+<div class="card">
+  <div class="item_show__header">
+    <h2 class="item_show__header-title">
+      <%= t ".title" %>
+      <% if allowed_to?(:create, :template) %>
+        <%= link_to t("actions.new_template", scope: "decidim.admin.templates"), [:new, :proposal_answer_template], class: "button button__sm button__secondary new" %>
+      <% end %>
+    </h2>
+  </div>
+  <% if @templates.any? %>
+    <div class="table-scroll">
+      <table class="table-list proposal_answer-templates">
```

Comment:
Is this class actually used? Or only for the spec (`./decidim-templates/spec/system/admin/admin_manages_proposal_answer_templates_spec.rb`)? Maybe we can refactor that so it doesn't use this class? 

```suggestion
      <table class="table-list">
```